pipelines/fbc-release/README.md

```diff
@@ -30,6 +30,10 @@ Tekton release pipeline to interact with FBC Pipeline
 
 ## Changelog
 
+### Changes since 0.22
```

Comment:
```suggestion
### Changes since 0.22.0
```

Reply:
Done, but please provide input in the [discussion below](https://github.com/redhat-appstudio/release-service-catalog/pull/212#discussion_r1336245671) whenever possible. Thanks!